.github/workflows/integration_tests_validation.yaml

```diff
@@ -36,9 +36,14 @@ jobs:
         uses: Swatinem/rust-cache@v1
       - run: sudo apt-get -y update
       - run: sudo apt-get install -y pkg-config libsystemd-dev libdbus-glib-1-dev libelf-dev libseccomp-dev
+      - name: Install runc 1.1.0
```

Comment:
Why do we need to install this version of runc?

Reply:
runc 1.1.0 changed how it handles `--work-dir` (or `--work-path`). Before 1.1.0 runc would create a directory called `criu-work` if the user does not specify a working directory. CRIU's default behaviour is to put the log and statistics files in the image path if work dir is not set. With runc 1.1.0 runc behaves like CRIU. That is also what I did for this PR and therefore the tests do not pass with older versions of runc.